src/router.ts

```diff
@@ -728,84 +724,89 @@ const routeVoterToSlackChannel = async (
+      `*Operator:* Voter *${userId}* was routed from *${adminCommandParams.previousSlackChannelName}* back to this channel by *${adminCommandParams.routingSlackUserName}*. See their thread with *${twilioPhoneNumber}* above.`,
+      { channel: destinationSlackChannelId }
+    );
 
-  await SlackApiUtil.sendMessage(
-    `*Operator:* Voter *${userId}* was routed from *${adminCommandParams.previousSlackChannelName}* back to this channel by *${adminCommandParams.routingSlackUserName}*. See their thread with *${twilioPhoneNumber}* above.`,
-    { channel: destinationSlackChannelId }
-  );
+    const timestampOfLastMessageInThread = await DbApiUtil.getTimestampOfLastMessageInThread(
+      userInfo[destinationSlackChannelId]
+    );
 
-  const timestampOfLastMessageInThread = await DbApiUtil.getTimestampOfLastMessageInThread(
-    userInfo[destinationSlackChannelId]
-  );
+    logger.debug(
+      `timestampOfLastMessageInThread: ${timestampOfLastMessageInThread}`
+    );
 
-  logger.debug(
-    `timestampOfLastMessageInThread: ${timestampOfLastMessageInThread}`
-  );
+    // Set destination thread to have same needs_attention status as origin thread
+    await DbApiUtil.setThreadNeedsAttentionToDb(
+      userInfo[destinationSlackChannelId],
+      destinationSlackChannelId,
+      needsAttention
+    );
 
-  // Set destination thread to have same needs_attention status as origin thread
-  await DbApiUtil.setThreadNeedsAttentionToDb(
-    userInfo[destinationSlackChannelId],
-    destinationSlackChannelId,
-    needsAttention
-  );
+    await SlackApiUtil.sendMessage(
+      `*Operator:* Voter *${userId}* was routed from *${adminCommandParams.previousSlackChannelName}* back to this thread by *${adminCommandParams.routingSlackUserName}*. Messages sent here will again relay to the voter.`,
+      {
+        channel: destinationSlackChannelId,
+        parentMessageTs: userInfo[destinationSlackChannelId],
+      }
+    );
 
-  await SlackApiUtil.sendMessage(
-    `*Operator:* Voter *${userId}* was routed from *${adminCommandParams.previousSlackChannelName}* back to this thread by *${adminCommandParams.routingSlackUserName}*. Messages sent here will again relay to the voter.`,
-    {
-      channel: destinationSlackChannelId,
-      parentMessageTs: userInfo[destinationSlackChannelId],
-    }
-  );
+    await routeVoterToSlackChannelHelper(
+      userInfo,
+      redisClient,
+      twilioPhoneNumber,
+      {
+        destinationSlackChannelName,
+        destinationSlackChannelId,
+        destinationSlackParentMessageTs: userInfo[destinationSlackChannelId],
+      },
+      timestampOfLastMessageInThread
+    );
+  }
 
-  await routeVoterToSlackChannelHelper(
-    userInfo,
-    redisClient,
-    twilioPhoneNumber,
-    {
-      destinationSlackChannelName,
-      destinationSlackChannelId,
-      destinationSlackParentMessageTs: userInfo[destinationSlackChannelId],
-    },
-    timestampOfLastMessageInThread
+  await DbApiUtil.setThreadRoutedToDb(
```

Comment:
the differ is freaking out here but can you confirm the only change is to move setThreadRoutedToDb lower?

Reply:
Yep!